### PR TITLE
New block ID logic

### PR DESCRIPTION
**Pull request by ivan-ristovic**

### CODE REVIEW

**Comment by ivan-ristovic** [comment]
[Block ID](https://trello.com/c/Ua3cThT8/7-block-id)

File: src/bbe/Program.java [hunk 1]
```diff
@@ -64,9 +65,10 @@ public static void main(String[] args)
 	        }
 	    }
 
-		// Logger.logInfo("Traversing dest tree.");
-		// traverser.traverseDestTree(vars);
+		Logger.logInfo("\n\n---- Traversing dest tree... ---");
```
Comment (milana-kovacevic) [inline src/bbe/Program.java]: Why 4 dashes?

**Review by milana-kovacevic — APPROVED** [review]
Using bit operators are in my opinion a bit confusing, but it makes sense to use them in this case.
Apart from that, everything seems legit. Approved.

File: src/bbe/Program.java (hunk 1 above)
Comment (ivan-ristovic) [reply src/bbe/Program.java]: Accident

**Pull request merged**